plugins/modules/aci_access_span_filter_group_entry.py

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Akini Ross <akinross@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_access_span_filter_group_entry
+short_description: Manage Access SPAN filter group entries (span:FilterEntry)
+description:
+- Manage Access SPAN filter group entries on Cisco ACI fabrics.
+options:
+  filter_group:
+    description:
+    - The name of the Access SPAN filter group.
+    type: str
+  source_ip:
+    description:
+    - The source IP Prefix.
+    type: str
+  destination_ip:
+    description:
+    - The destination IP Prefix.
+    type: str
+  first_src_port:
+    description:
+    - The first source port (from port).
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ unspecified, dns, ftp, http, https, pop3, rtsp, smtp, ssh ]
+  last_src_port:
+    description:
+    - The last source port (to port).
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ unspecified, dns, ftp, http, https, pop3, rtsp, smtp, ssh ]
+  first_dest_port:
+    description:
+    - The first destination port (from port).
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ unspecified, dns, ftp, http, https, pop3, rtsp, smtp, ssh ]
+  last_dest_port:
+    description:
+    - The last destination port (to port).
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ unspecified, dns, ftp, http, https, pop3, rtsp, smtp, ssh ]
+  ip_protocol:
+    description:
+    - The IP Protocol.
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ unspecified, egp, eigrp, icmp, icmpv6, igmp, igp, l2tp, ospfigp, pim, tcp, udp ]
```

Comment:
is it limited to those or can you input any protocol number?

Reply:
same applies as the port number, resolving this and tracking on comments there.